django_outbox_pattern/decorators.py

```diff
@@ -17,17 +18,24 @@ class Config(NamedTuple):
 
 
 def publish(configs: List[Config]):
-    def save(self, *args, **kwargs):
-        with transaction.atomic():
-            super(self.__class__, self).save(*args, **kwargs)
-            for config in configs:
-                _create_published(self, *config)
-
-    def decorator_publish(cls):
-        cls.save = save
-        return cls
-
-    return decorator_publish
+    def wrapper(cls):
+        class PublishModel(cls):
+            def save(self, *args, stop=False, **kwargs):
```

Comment:
suggestion: Change the `stop` to more clarify parameter, for example `not_publish_message_to_queue`. What do you think about this?
One docstring here joined with the documentation can be useful.

Reply:
Amazing sugestion

Reply:
even though this suggestion would bring more readability, being sucint while naming function arguments sounds a good practice. if you really like to rename this arg, maybe you could  call it `send_to_queue`, with default `True` value.
This would not break the reading of the method with a negative condition. What do you guys think on this?